nemo/collections/nlp/parts/utils_funcs.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2020 NVIDIA. All Rights Reserved.
+# Copyright 2018 The Google AI Language Team Authors and
+# The HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+__all__ = ['list2str']
+
+
+def list2str(l):
```

Comment:
add docstring

Reply:
This function is not used by the user. Do we need docstring for all function?

Reply:
its always good to add some comments or docstring, but yeah this case is less important than a user facing api